server/src/main/java/org/opensearch/discovery/InitializeExtensionsRequest.java

```diff
@@ -56,21 +61,31 @@ public DiscoveryNode getSourceNode() {
         return sourceNode;
     }
 
+    public int getPort() {
```

Comment:
(Suggestion) Might consider naming this `getOpenSearchPort()`. While it can't be anything else in this context, when accessed remotely from an extension, this shows up as `extensionInitRequest.getPort();` and in that context, "port" is less clear as there is also an extension port.

Reply:
Sure. Will take care of it in the next PR.